### DISCUSSION TITLE
How to see properties of scheduled job?

### DISCUSSION THREAD

**Discussion by llamafilm** [netbox-community/netbox#14989] [Q&A]
If I setup a script to run on a recurring schedule, how can I see what that schedule is?
The `/extras/scripts/results/768/` page only shows the next time it will run, but I would like to know its interval, i.e. weekly or monthly, etc.

Also — how can I see the variables chosen for this job?  For example if I run the same script on 2 different schedules, each with different variables.

**Comment by Alex-Ignatov** [comment]
From here you can see schedules: http://localhost:8000/core/jobs/
And from here http://localhost:8000/core/background-queues/0/scheduled/ you can see parameters that were assigned to each.

But I agree it would have been much better if there was a unified view for the scheduled scripts. Since a single script can be scheduled for each site as input parameter and all sites can be on different schedule/offset based on time/location and it is not easy to manage this.
